3.resistance-signature/scripts/nbconverted/8.1. last-batch-compile-otherclone-datase.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Compile dataset of clones resistant to other drugs
+# 
+# **Gregory Way, 2021**
+# 
+# **Yu Han, 2021**
+# 
+# This script is modified from Greg Way's original scripts of 8.compile-otherclone-dataset.
+# 
+# This dataset includes new batches of 24~27 including WT (10, 12-15) and Bz-resistant (6-10) clones.
+# They are DMSO-treated clones to later apply the bortezomib resistance signature.
+
+# In[1]:
+
+
+import sys
+import pathlib
+import numpy as np
+import pandas as pd
+
+from pycytominer.cyto_utils import infer_cp_features
+
+sys.path.insert(0, "../2.describe-data/scripts")
+from processing_utils import load_data
+
+
+# In[2]:
+
+
+np.random.seed(1234)
+
+
+# In[3]:
+
+
+data_dir = pathlib.Path("../0.generate-profiles/profiles")
+cell_count_dir = pathlib.Path("../0.generate-profiles/cell_counts/")
+
+output_dir = pathlib.Path("data")
+
+profile_suffix = "normalized.csv.gz"
+
+
+# In[4]:
+
+
+datasets_val = {
+        "2021_08_02_Batch24": ["221057"],
+        "2021_08_02_Batch25": ["221058"],
+        "2021_08_03_Batch26": ["221093"],
+        "2021_08_03_Batch27": ["221094"],
+}
+
+
+# In[5]:
+
+
+#added 'val' to each original variable names from 8.0
+
+full_df_val = []
+for dataset_val in datasets_val:
+    dataset_df_val = []
```

Comment:
I don't think that you need this loop here... all you need is to loop over the batches (as you've done in line 65)

Reply:
The dataset_val was used to define the `Metadata_dataset=dataset_val` in the next few lines, so I kept it here just so the following up codes won't be messed up.